packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -207,9 +207,42 @@ class FlutterTask extends DefaultTask {
         }
     }
 
+    FileCollection readDependencies(File dependenciesFile) {
+        if (dependenciesFile.exists()) {
+            try {
+                // Dependencies file has Makefile syntax:
+                //   <target> <files>: <source> <files> <separated> <by> <space>
+                String depText = dependenciesFile.text
```

Comment:
It's technically in 'Makefile' format, which does allow comments. However, gen_snapshot only emits one line containing 'target: sources'.